README.md

```diff
@@ -31,6 +31,15 @@ The build outputs will be available in the directory `build/`.
 
 ## Flashing the firmware
 
+### Usefull info
+* Don’t cycle the EPM and you cannot get zapped
+* Don't sap your computer it will not survive 
+* HV section to avoid touching during debug, see fig(1)
+* Solder Jumper settings see fig(1)
+
+<img src="annotation.jpg" class="thumbnail" title="fig(1)">
```

Comment:
Markdown does not support this syntax, it is only valid for Zubax Docs. You probably meant this:

`![Figure 1](annotation.jpg "Figure 1")`